src/telemetry_util.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <format>
+#include <string_view>
+
+#include "string_util.h"
+
+extern "C" {
+#include <ngx_http.h>
+}
+
+namespace datadog::nginx {
+
+constexpr std::string_view relative_filepath(
+    std::string_view absolute_filepath) {
+  constexpr std::string_view prefix = "nginx-datadog";
+  size_t pos = absolute_filepath.find(prefix);
+  return pos != std::string_view::npos ? absolute_filepath.substr(pos)
+                                       : absolute_filepath;
```

Comment:
The problem with this method is, if nginx-datadog is not in the string (which I agree with you should not happen), we will leak the full_patch, can we add a safe guard please ?

Reply:
Could you clarify what the potential risks are of exposing the full path? (which is _highly_ improbable unless the repo change name or the ci building the project is cloning the repo under another name). Also, could you elaborate on what you mean by safeguard? Do you have any suggestions or proposals in mind?

Reply:
For instance, i have my name.surname in my full_path, which is PII. In PHP, I did this:
https://github.com/DataDog/dd-trace-php/blob/0e702530af821f358ccd1dbdecd9535728eed459/ext/telemetry.c#L50

I agree that is a bit overkill and I could accept to not include that it does not add runtime overhead so I think "prevention is better than cure" is the way to go.

Reply:
Interesting 🤔 Make sense. I'll adjust the method to return only the name of the file instead of the full path. Thanks for raising your concerns.

Reply:
addressed in e5d20039095d87d1e576fbd203f823d0c8af0357